sycl/doc/extensions/experimental/sycl_ext_codeplay_kernel_fusion.asciidoc

```diff
@@ -48,6 +48,16 @@ or may be in a preliminary state.  The specification itself may also change in
 incompatible ways before it is finalized.  *Shipping software products should
 not rely on APIs defined in this specification.*
 
+[NOTE]
+====
+There is a link:../proposed/sycl_ext_oneapi_graph_fusion.asciidoc[follow-up
+proposal] for fusion based on the SYCL graph API. That proposal continues some
```

Comment:
NIT: link the SYCL Graphs proposal too

Reply:
As the SYCL graphs proposal has not yet been merged, there's not yet a stable URL that we could link to.

Reply:
Added the link to the PR (and a note on my to-do list to update later).